CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ New Features
 
 - `reconnecting` and `reconnected` events on Room and LocalParticipant are now fired asynchronously. (JSDK-2696)
 - twilio-video.js now raises `connect()` errors due to network disruptions quicker by not retrying after the first connection attempt fails. (JSDK-2682)
+- twilio-video.js now uses server supplied timeout value to determine reconnect attempts in case of loss of signaling connection.
```

Comment:
@makarandp0 Please ask @ceaglest to verify the language here, since he came up with the proposal. Also, please add the ticket number.

Reply:
We didn't have language in our CL around the server timeout, because we had hardcoded 30 seconds in previous betas. Here is my suggestion:

```suggestion
- twilio-video.js only attempts to reconnect to a Room while the Participant's session is valid (typically 30 seconds after entering the reconnecting state) instead of using a fixed number of retries.
```